### PR TITLE
Corrections/passing master

### PR DESCRIPTION
**Pull request by schultzh**
The goal of this PR is to fix all linting errors in the master branch which will allow students to merge to master with confidence going forward knowing that master was and is still passing the build.

### CODE REVIEW

**Comment by schultzh** [comment]
In connection with Issue #56.

**Comment by schultzh** [comment]
@koscinskic The remaining issues are related to failing test cases regarding data collection. Could you and the software arc team look address those failing test cases please? I'd love to get is passing for tomorrow's lab!

**Comment by MaddyKapfhammer** [comment]
Looks good to me @schultzh except for the failures associated with the failing test cases regarding data collection. I will approve this PR once those failures are fixed. I agree that it would be beneficial for this to be completed by the lab session tomorrow!

**Comment by koscinskic** [comment]
The build _is_ passing now, but only because half of the test cases are marked as xfail. I believe this is acceptable so long as we are moving to change those to normal cases. All that is necessary for the issues case is the Travis variable (which we have covered), and I believe the other two are simply lacking the respective pull request that they belong to. Otherwise, we should merge this as soon as possible.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/hannahschultz/.local/share/virtualenvs/cogitate_tool-HEnOOtpD/bin/python"
+}
```
Comment (enpuyou) [inline .vscode/settings.json]: This was being raised in another PR as well: should this file go into the master?

**Comment by schultzh** [comment]
@koscinskic I don't think writing XFAIL test cases is the best way to deal with this just to get master to pass. I have added the environment variable token to my branch that will address those test cases.

File: .vscode/settings.json (hunk 1 above)
Comment (schultzh) [reply .vscode/settings.json]: I am happy to remove it since it includes only a local path from my device.

**Comment by koscinskic** [comment]
Update on the cases marked with xfail:

test_collect_commits_user_email_key is marked because the source code for the corresponding case was lost at some indeterminate point. This should be resolved by a pull request initiated by the associated team.

test_retrieve_issue_data_retrieves_issues is marked as the case is experiencing connection issues due to the sheer volume of traffic (I'm assuming this is the cause). The case DOES pass given enough time, but is marked as a failure if the allotted time is overwritten. This warrants its own pull request and won't be handled here.

**Comment by schultzh** [comment]
@koscinskic Is this ready to merge?

**Pull request merged**